src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
+}
```

Comment:
No needed to separate Lottery and ColorSupplier classes

Reply:
public class Lottery  {
    private static final int COLORS_NUMBER = Colors.values().length ;
    private static final int BALL_NUMBER = 100;

    public Ball getRandomBall() {
        Ball ball = new Ball();
        ball.setBallColor(getRandomColor());
        ball.setBallNumber(new Random().nextInt(BALL_NUMBER));
        return ball;
    }
    public String getRandomColor() {
        int index = new Random().nextInt(COLORS_NUMBER);
        return Colors.values()[index].toString();
    }
}